### ISSUE TITLE
Schema documentation does not show marginal note and correction status

### ISSUE THREAD

**Issue by adamconder** [UKHomeOffice/lev-api-docs#18]
- https://github.com/UKHomeOffice/lev-api-docs/blob/master/lev-api.md states that:

``` json
"required": [
          "blockedRegistration",
          "cancelled",
          "correction",
          "potentiallyFictitiousBirth",
          "marginalNote",
          "reRegistered"
        ]
```

`marginalNote` and `correction` is required yet the schema is missing these values. 

Also, the schema is showing these additional properties within `status` but they're not shown in the required list. Can you clarify these please?

``` json
"cautionMark": {
            "type": "boolean",
            "description": "TBD, what does this mean",
            "default": false
          },
          "courtOrder": {
            "type": "string",
            "example": "None",
            "description": "TBD, what does this mean",
            "enum": [
              "None",
              "In place",
              "Revoked"
            ]
          }
```

It would also be good if you could expand on `"description": "TBD, what does this mean"` to provide some context as to what these flags indicate.

**Comment by timgent** [comment]
Thanks for flagging this. Think some additional work is needed here as follows (will use this issue to track):
1) Update docs so only following status flag is required:
- blockedRegistration

2) Update status fields so there should be only:
blockedRegistration: Boolean
cancelled: Option[Boolean]
potentiallyFictitiousBirth: Option[Boolean]
reRegistered: Option[String]
correction: Option[String]
marginalNote: Option[String]

3) Add proper descriptions for each of the status fields

**Comment by timgent** [comment]
@adamconder Currently we have blocked registrations where all the other fields, including the other status fields, are blanked out, leaving just the names and system number. This makes more of those status fields optional and maybe reduces the usefulness of the schema.

Does this work for you as a way of handling blocked registrations or can you think of any other ways to handle it that would work better?

**Comment by timgent** [comment]
PR created here:
https://github.com/UKHomeOffice/lev-api-docs/pull/23

**Comment by adamconder** [comment]
> Currently we have blocked registrations where all the other fields, including the other status fields, are blanked out, leaving just the names and system number. This makes more of those status fields optional and maybe reduces the usefulness of the schema.

@timgent Could you please clarify what you mean by this I don't understand your response.

**Comment by timgent** [comment]
So if you get a blocked record it will look something like this:

``` json
{
  "location": {

  },
  "subjects": {
    "child": {
      "name": {

      },
      "originalName": {

      }
    },
    "father": {
      "name": {

      }
    },
    "mother": {
      "name": {

      }
    },
    "informant": {
      "name": {

      }
    }
  },
  "systemNumber": 123456789,
  "id": 123456789,
  "status": {
    "blockedRegistration": true
  },
  "previousRegistration": {

  }
}
```

**Comment by timgent** [comment]
So you get empty objects for all the required fields, system number is returned, but nothing else comes back

**Comment by adamconder** [comment]
@timgent Thank you for clarifying this. This is helpful in understanding the variations of data coming back from GRO. Can there be a situation that arises where there is multiple status flags being returned. This is an object on the schema so doesn't indicate if multiple can occur?

**Comment by timgent** [comment]
Yes multiple flags can be returned. However if you have blockedRegistration flag come back you won't get any information about the others
